src/Item.h

```diff
@@ -200,6 +201,7 @@ class cItem
 	int            m_RepairCost;
 	cFireworkItem  m_FireworkItem;
 	cColor         m_ItemColor;
+	Json::Value    m_BlockEntityTag;
```

Comment:
There was a decision https://forum.cuberite.org/thread-3325-lastpost.html to add proper Item tag support to cItem, once that's done we can rebase this PR on top.